iothub/service/src/Common/Exceptions/ErrorCode.cs

```diff
@@ -219,8 +220,8 @@ public enum ErrorCode
         /// <summary>
         /// IoT hub ran into a server side issue.
         /// There can be a number of causes for a 500xxx error response. In all cases, the issue is most likely transient.
-        /// IoT hub nodes can occasionally experience transient faults. When your device tries to connect to a node that is
-        /// having issues, you receive this error. To mitigate 500xxx errors, issue a retry from the device.
+        /// IoT hub nodes can occasionally experience transient faults. When your application tries to connect to a node that is
+        /// having issues, you receive this error. To mitigate 500xxx errors, issue a retry from your application.
```

Comment:
this error is returned to the service-side application, which isn't associated with a particular device identity.